src/content/peerconnection/audio/js/main.js

```diff
@@ -356,11 +360,12 @@ window.setInterval(() => {
           // append to chart
           bitrateSeries.addPoint(now, bitrate);
           headerrateSeries.addPoint(now, headerrate);
-          bitrateGraph.setDataSeries([bitrateSeries, headerrateSeries]);
+          targetBitrateSeries.addPoint(now, report.targetBitrate);
+          bitrateGraph.setDataSeries([bitrateSeries, headerrateSeries, targetBitrateSeries]);
```

Comment:
when running on pre-bitrate-stat browsers: will addPoint(now, null) work fine and do nothing?

Reply:
Yes. The graph library interprets "undefined" as "no value" and does not draw it.